src/lib/components/Layout/Col.svelte

```diff
@@ -0,0 +1,38 @@
+﻿<div {...props}>
+  <slot />
+</div>
+
+<script lang="ts">
+  type colSpan = number | string | undefined;
+
+  export let col: colSpan = undefined;
+  export let sm: colSpan = undefined;
+  export let md: colSpan = undefined;
+  export let lg: colSpan = undefined;
+
+  export let fill: boolean = false;
+  export let top: boolean = false;
+  export let middle: boolean = false;
+  export let bottom: boolean = false;
+
+  const breakpoints = ['col', 'sm', 'md', 'lg'];
+
+  const breakpointClasses = [col, sm, md, lg].map((value, i) => {
+    const breakpoint = breakpoints[i];
+    return value === undefined
+      ? ''
+      : breakpoint + '-' + value;
+  });
+
+  const utilities = ['col-fill', 'align-top', 'align-middle', 'align-bottom'];
+
+  const utilityClasses = [fill, top, middle, bottom].map(
+    (value, i) => value ? utilities[i] : ''
+  );
+
+  const props = {
+    ...$$restProps,
+    class: [$$restProps.class, 'col', ...breakpointClasses, ...utilityClasses]
+      .join(' ')
+  };
```

Comment:
```suggestion
$: classes = `col ${$$restProps.class ?? ''} ${computeClasses(
    'col', { col, sm, md, lg }
  )}`;
```

There's an helper function in utils.ts to manage complexe classes that depends on props
Don't forget to add the import at the top
`import { computeClasses } from '../../utils';`
You will need to add the following in  `propToClass` object in utils.ts:

```
  col: (pf, val) => `${pf}-${val}`,
  sm: (_pf, val) => `sm-${val}`,
  md: (_pf, val) => `md-${val}`,
  lg: (_pf, val) => `lg-${val}`,
```